settings.gradle

```diff
@@ -1,3 +1,4 @@
+include ':articles-aem-renderer'
```

Comment:
I don't think this module actually exists, so this line can be removed :)

Reply:
For some reason when I remove it.  it can find my model
![godtools____documents_github_godtools-android__-_____settings_gradle__godtools_](https://user-images.githubusercontent.com/970048/44482293-e9182d80-a615-11e8-998e-df0ee029a269.png)

Reply:
hmm, interesting. leave it for now, I'll take a look later

Reply:
OK I figured it out the .iml was misconfigured.

Reply:
makes sense